### PR TITLE
Portfolio: Accessibility

### PR DESCRIPTION
**Pull request by stenlisuryadinata**
all checked report listing of all points that I have evaluated.

https://github.com/LarryIVC/portfolio/issues/14

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 </head>
 <body>
     <header>
-        <div class="my-header">
+        <div class="my-headers">
             <a class="logo" href="index.html">Larry Villegas</a>
```
Comment (stenlisuryadinata) [inline index.html]: the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low

**Review by stenlisuryadinata — COMMENTED** [review]
the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low

**Comment by LarryIVC** [comment]
### **In this PR, I did the following:**

- [x] Create a new branch to Solve Accessibility problems.
- [x]  Correct the low contrast in the logo.
- [x]  Test Linters in the project.

### **Report accessability :**

all checked points report listing, as a result of my evaluation, are as follows: 

Page titles - no issue found
image text alternatives - no issue found
text headings - no issue found 
color contrast - have an issue, please check below
resize - no issue found
interaction - no issue found 
moving content - no issue found 
multimedia - no issue found 
the basic structure of the page - no issue found. 

everything is good, the only remarks are as follows: 

1. the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L16

2. the let's connect contrast (2.95:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L38

4. the back-end contrast (3.68:1) is low
the contrast between text and background colors is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L58

5. the HTML, CSS, and javascript text  contrast (3.39:1) is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L64

6. the contact form accessibility is  Missing the form label
 the contact form does not have a corresponding label for a form control 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L196

7. the text contrast (3.39:1) is low
the contrast between text and background colors is low. 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L192

8. the text area missing form label
the text area does not have a corresponding label for form control 

Thank you

Ps: as per the mentor guideline adding the summary to the pull request

**Review by jasonfelice — COMMENTED** [review]
Hi @LarryIVC, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't has contributions from all collaborators.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by LarryIVC** [comment]
Hi, @jasonfelice as you can see in the following image:
image
The two members and collaborators of the project are participating.

**Comment by stenlisuryadinata** [comment]
all checked points report listing, as a result of my evaluation, are as follows: 

Page titles - no issue found
image text alternatives - no issue found
text headings - no issue found 
color contrast - have an issue, please check below
resize - no issue found
interaction - no issue found 
moving content - no issue found 
multimedia - no issue found 
the basic structure of the page - no issue found. 

everything is good, the only remarks are as follows: 

1. the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L16

2. the let's connect contrast (2.95:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L38

4. the back-end contrast (3.68:1) is low
the contrast between text and background colors is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L58

5. the HTML, CSS, and javascript text  contrast (3.39:1) is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L64

6. the contact form accessibility is  Missing the form label
 the contact form does not have a corresponding label for a form control 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L196

7. the text contrast (3.39:1) is low
the contrast between text and background colors is low. 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L192

8. the text area missing form label
the text area does not have a corresponding label for form control 

Thank you

Ps: as per the mentor guideline adding the summary to the pull request

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @stenlisuryadinata  and @LarryIVC 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


# Highlights
- No linter errors ✔️
- Good implementations ✔️


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

__check the comments for details

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**